autoload/snipe/core.vim

```diff
@@ -168,6 +171,9 @@ function! s:GetJumpCol(jump_tree) " {{{
   for [jump_seq, jump_col] in jump_items
     " this loop builds the highlighted line, adding highlights from left to right;
     " previous multi-token jump sequences are accounted for by tracking col_offset.
+    if !g:snipe_display_full_jump_seq
+        let jump_seq = jump_seq[0]
+    endif
     let len_jump_seq = strlen(jump_seq)
```

Comment:
Can we use a new variable instead of reassigning `jump_seq`?

***

We should also look for simplifications in the subsequent code:

* `len_jump_seq` will always be `1` (do we even need it now?)
* do we still need to track `col_offset`?
* can we get rid of `hl_cols` and just `call add(hl_ids, matchaddpos(g:snipe_hl1_group, [[lnum, jump_seq]]))`?